external/loaders/loaders/mappers/_fine_resolution_budget.py

```diff
@@ -427,17 +427,17 @@ def open_fine_res_apparent_sources(
         drop_vars (sequence): optional list of variable names to drop from dataset
     """
 
-    # use default which is valid for real data
-    if rename_vars is None:
-        rename_vars = {
-            "grid_xt": "x",
-            "grid_yt": "y",
-            "pfull": "z",
-            "delp": "pressure_thickness_of_atmospheric_layer",
-        }
-
-    fine_resolution_sources_mapper: GeoMapper = FineResolutionSources(
-        open_fine_resolution_budget(fine_res_url),
+    rename_vars: Mapping[Hashable, Hashable] = {
```

Comment:
I'm under the impression these are never used apart from their defaults. Please let me know if that's not the case.

Reply:
Yeah, it's a good bet that this is tied pretty concretely to the underlying data, which I assume is mostly static for now.